cns/saas/admin_guide/start/enforcer/reqs.adoc

```diff
@@ -195,14 +165,138 @@ When you Deploy the Enforcer on Kubernetes, OpenShift, and Linux hosts, the Linu
+* **Service mesh:** The Enforcer requires Istio 1.8 and above.
+
+To deploy the Enforcer on GKE, you must have https://cloud.google.com/kubernetes-engine/docs/how-to/iam#predefined[Kubernetes Engine Admin] permissions. The Enforcer ignores Fargate and other serverless workloads in your cluster.
+
+==== Supported Orchestrators
+
+The Enforcer supports the following orchestrators.
+
+===== Google Kubernetes Engine (GKE)
+
+* Supported release: `1.21.6`
+    ** Supported images:
+
+        *** `cos_containerd`
+        *** `ubuntu`
+        *** `ubuntu_containerd`
+
+    ** Tested using:
+
+        *** Docker `20.10.3`
+        *** Kubelet `1.21.6-gke.1500`
+        *** Kernel version `5.4.144+`
+        *** Calico CNI
+
+* Supported release: `1.18.16`
+    
+    ** Supported images:
+        
+        *** `cos_containerd`
+        *** `ubuntu`
+        *** `ubuntu_containerd`
+
+    ** Tested using: 
+        
+        *** Docker `19.3.14`
+        *** Kubelet `v1.18.16-gke.502`
+        *** Istio `1.6.14-gke.0`
+
+===== Amazon Elastic Kubernetes Service (EKS)
+
+* Supported release: `1.21.5-eks-9017834`
+    
+    ** Supported image: Amazon Linux 2
+    
+    ** Tested using:
+        
+        *** Kernel version `5.4.176-91.338.amzn2.x86_64`
+        *** Docker `20.10.7`
+        *** Kubelet `v1.21.5-eks-9017834`
+        *** Amazon VPC CNI
+
+===== Azure Kubernetes Service (AKS)
+
+* Supported release: `1.21.7`
+    
+    ** Supported image: `Ubuntu 18.04.6 LT`
+    
+    ** Tested using:
+
+        *** Containerd `1.4.9+azure`
+        *** Kernel `5.4.0-1067-azure`
+        *** Azure CNI
+
+===== OpenShift Container Platform (OCP)
+
+* Supported release: `4.9.27`
+
+    ** Supported image: `Red Hat Enterprise Linux (RHEL) CoreOS 49.84.202203221419-0 (Ootpa)`
+
+    ** Tested using: 
+
+        *** Kernel `4.18.0-305.40.2.el8_4.x86_64`
+        *** CRI-O `1.22.3-2.rhaos4.9.git28c5a70.el8`
+        *** Kubelet `v1.22.5+5c84e52`
+        *** CNI: OpenShiftSDN
+
+* Supported release: `4.8.35 `
+
+    ** Supported image: `RHEL CoreOS 48.84.202201241104-0 (Ootpa)`
+
+    ** Tested using: 
+
+        *** Kernel `4.18.0-305.34.2.el8_4.x86_64`
+        *** CRI-O `1.21.4-9.rhaos4.8.gitaebb17b.el8`
+        *** Kubelet `v1.21.6+bb8d50a`
+        *** CNI: OVN-Kubernetes
+
+* Supported release: `4.7.42`
+
+    ** Supported image: `RHEL CoreOS 47.83.202107210005-0 (Ootpa)`
+    
+    ** Tested using: 
+        
+        *** Kernel `4.18.0-240.23.2.el8_3.x86_64`
+        *** CRI-O `1.20.4-4.rhaos4.7.gitf7276ed.el8`
+        *** CNI: `OpenShiftSDN`
+        *** Kubelet `v1.20.0+558d959`
+
+===== Tanzu (TKGI)
+
+* Supported release: `1.8`
+* Kubernetes `1.16` or later
+* Customer-managed: on-prem or cloud 
+* Provider-managed platform: AKS, EKS, and GKE
+* OpenShift Container Platform `4.6-4.8`
```

Comment:
This should be changed to 4.7, 4.8, and 4.9

Reply:
Updated accordingly.